hack/preload-images/generate.go

```diff
@@ -128,6 +132,20 @@ func generateTarball(kubernetesVersion, containerRuntime, tarballFilename string
 	return copyTarballToHost(tarballFilename)
 }
 
+func verifyStorage(containerRuntime string) error {
+	if containerRuntime == "docker" || containerRuntime == "containerd" {
+		if err := verifyDockerStorage(); err != nil {
+			return errors.Wrap(err, "Docker storage type is incompatible")
+		}
+	}
+	if containerRuntime == "cri-o" {
+		if err := verifyPodmanStorage(); err != nil {
```

Comment:
I dont see this funciton defined, and I dont thtink this is needed, this is supposed to verify the Outter Docker Storage driver (docker driver is used to generate images in our CI we dont use podman) so we can delete this.

Reply:
Are you sure ? The tar commands seems to be running properly in the docker, so it needs to make sure that the /var files that it is archiving is for the right storage driver. The images we are archiving for cri-o, is using same storage as podman.